packages/react-native-attestation/package.json

```diff
@@ -2,41 +2,28 @@
   "name": "@hyperledger/aries-react-native-attestation",
   "version": "1.0.0",
   "description": "Mobile app attestation",
-  "main": "lib/commonjs/index",
-  "module": "lib/module/index",
-  "types": "lib/typescript/index.d.ts",
-  "react-native": "src/index",
-  "source": "src/index",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "module": "build/index.ts",
+  "source": "src/index.ts",
   "files": [
     "src",
-    "lib",
-    "android",
-    "ios",
-    "cpp",
-    "*.podspec",
```

Comment:
@jleach I notice we don't have `lib` or `cpp` anymore, are we good without those?